packages/runner-ct/webpack.config.ts

```diff
@@ -80,10 +80,6 @@ const config: webpack.Configuration = {
 config.plugins = [
   // @ts-ignore
   ...config.plugins,
-  new HtmlWebpackPlugin({
```

Comment:
We no longer need an entry point to load ct tests?

Reply:
Nope - this is only used when developing `runner-ct`, which we do not do anymore. We now get the index file here, for app: https://github.com/cypress-io/cypress/blob/10.0-release/packages/data-context/src/sources/HtmlDataSource.ts#L60

Reply:
Just curious, does this mean we no longer need the `HtmlWebpackPlugin` for the runner package either?

Reply:
I'd say so. I deleted it.